mpas_analysis/analysis_task_template.py

```diff
@@ -33,6 +33,8 @@
 Xylar Asay-Davis
 '''
 
+from __future__ import absolute_import, division, print_function, unicode_literals
```

Comment:
These lines are all too long (more than 80 characters) to be PEP8 compliant.  Could they be broken into multiple lines, please?

Reply:
Can do! Are you trying to follow PEP8 exactly? I've got lots of warnings for variable naming capitalization so I turned off the PEP8 checks.

Reply:
I've taken care of these already, so no need to change them again.

Reply:
We're trying to follow PEP8 pretty closely but we use mixed case for variable names.  This doesn't give me any trouble in spyder but I guess other checkers might be stricter about that particular issue.

We definitely allow longer lines when it doesn't make sense to break them but this would be a case where I would break up lines.